src/components/WebCLI/Output.js

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useMemo, useRef, useState } from "react";
+
+// Colors taken from the VSCode section of
+// https://en.wikipedia.org/wiki/ANSI_escape_code#3/4_bit
+const ansiColors = {
+  0: null, // reset
+  30: "0,0,0", // Black
+  31: "205,49,49", // Red
+  32: "13,188,121", // Green
+  33: "229,229,16", // Yellow
+  34: "26,114,200", //  Blue
+  35: "188,63,188", // Magenta
+  36: "17,168,205", // Cyan
+  37: "229,229,229", // White
+  90: "102,102,102", // Bright Black (Gray)
```

Comment:
Haha, Bright Black (Gray)

Reply:
Haha yeah I copied the description from the wikipedia article above, I also thought it was a funny inclusion :D